src/main/java/com/bulkbuy/enterprise/BulkBuyController.java

```diff
@@ -7,19 +7,29 @@
 public class BulkBuyController {
 
     /**
+     * @aurthor
```

Comment:
Since we can go through history in version control now, the @author tag is no longer used.
Also, in JavaDoc, when you add @ params, define them as well.